src/main/java/no/finn/unleash/FakeUnleash.java

```diff
@@ -24,6 +26,21 @@ public boolean isEnabled(String toggleName, boolean defaultSetting) {
         }
     }
 
+    @Override
+    public List<String> getToggleNames() {
+        return features.keySet().stream().collect(Collectors.toList());
+    }
+
+    @Override
+    public List<String> getEnabledToggleNames() {
+        return features.keySet().stream().filter(feature -> features.get(feature)).collect(Collectors.toList());
+    }
+
+    @Override
+    public List<String> getEnabledToggleNames(UnleashContext context) {
+        return getEnabledToggleNames();
+    }
+
```

Comment:
Missing test cases for the FakeUnleash. It's important to verify the behaviour of the FakeUnleash implementation, as many users relies on it to run their own unit-tests, independent of the unleash infrastructure.

Reply:
Thank you for the clarification of what this class is used for. It wasn't immediately obvious to me.